src/main/java/no/digipost/monitoring/thirdparty/NoResultTimedThirdPartyCall.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (C) Posten Norge AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package no.digipost.monitoring.thirdparty;
+
+import no.digipost.monitoring.micrometer.AppStatus;
+
+import java.util.Optional;
+import java.util.function.BiFunction;
+import java.util.function.Supplier;
+
+public class NoResultTimedThirdPartyCall extends TimedThirdPartyCall<NoResult> {
```

Comment:
Kan egentlig godt hende at det beste her er å ha `NoResultTimedThirdPartyCall` som en helt separat type fra `TimedThirdPartyCall`, og at man heller bruker composition i stedet for inheritance, altså at man assigner en `TimedThirdPartyCall<NoResult>` til et felt, også delegerer `public void call(Runnable thirdPartyCall)` til denne:
```java
    public void call(Runnable thirdPartyCall) {
        timedThirdPartyCall.call(() -> {
            thirdPartyCall.run();
            return NoResult.NO_RESULT;
        });
    }
```

Reply:
[064bfb1](https://github.com/digipost/digipost-micrometer-prometheus/pull/26/commits/064bfb102fb09c68570a41fde15cdc97553d701a)